app/Tquery/Config/TqConfig.php

```diff
@@ -152,4 +173,12 @@ private function addColumn(
             renderer: $renderer,
         );
     }
+
+    private static function assertType(TqDataTypeEnum|TqDictDef $type, bool $is, TqDataTypeEnum ...$types): void
```

Comment:
Trochę ciężko zrozumieć te parametry, zwłaszcza z tą bool flagą.

Może lepiej zrobić 2 metody:`assertTypeIn($type, $types)`  i `assertTypeNotIn`

Reply:
Z drugiej strony mają po 3 linijki więc nie wiem czy to aż taki problem, że powtórzona implementacja:P Ale dobra, nie będę się upierał tak przy prywatnej funkcji.